### ISSUE TITLE
Port on Issuer URL Authority

### ISSUE THREAD

**Issue by pringinacio** [MauroDataMapper-Plugins/mdm-plugin-authentication-openid-connect#16]
When Issuer URL Authority contains a port (example: **keycloak.org:8443**), exceptions are thrown due to the "unexpected" port presence on Authority,

A possible solution is to modify the file **grails-app/services/uk/ac/ox/softeng/maurodatamapper/plugins/authentication/openid/connect/authentication/OpenidConnectAuthenticationService.groovy** 

and replace 

`createdBy: "openidConnectAuthentication@${issuerUrl.authority}",   `                                                   

by 
`createdBy: "openidConnectAuthentication@${issuerUrl.host}",`

This fixes the issue, then authority contains a port.

Example to reproduce the problem:

```
  public static void main(String[] args) throws MalformedURLException {
    URL url = new URL("https://keycloak.org:8443/realms/maurodatamapper");

    System.out.println(url.getAuthority());
    System.out.println(url.getHost());
  }
```

The output is:
```
keycloak.org:8443
keycloak.org
```

**Comment by jamesrwelch** [comment]
Hi @pringinacio, Thank you very much for reporting and investigating the issue.  Your fix looks promising - would you like to submit it as a pull request which we can review and accept?

**Comment by pringinacio** [comment]
I've opened this PR: 

https://github.com/MauroDataMapper-Plugins/mdm-plugin-authentication-openid-connect/pull/17

**Comment by jamesrwelch** [comment]
Closed with merge of #17 - thanks!
